openedx/core/djangoapps/schedules/management/commands/__init__.py

```diff
@@ -45,6 +51,10 @@ def handle(self, *args, **options):
             num_days = (7 * num_weeks) + 1
             self.offsets = range(-7, -num_days, -7)
 
+        check_completion = options.get('check-completion')
```

Comment:
nit: the actual argument string gets converted to `'check_completion'`, like ['override_recipient_email'](https://github.com/edx/edx-platform/pull/23359/files#diff-37c56b22e888293e35425d1865526d78R67) below.

Reply:
👍